docs/events/index.md

```diff
@@ -0,0 +1,126 @@
+# Testplane Events
+
+## Overview {#overview}
+
+### How is events description made {#how-is-events-description-made}
+
+
+Below are all the events of Testplane that you can subscribe to in your plugin.
+
+The description of each event begins with tags, which are represented by the following options:
+
+* _sync_ or _async_ indicate, respectively, the synchronous and asynchronous modes of calling the event handler;
+* _master_ indicates that this event is available from the Tesplane master process;
+* _worker_ indicates that this event is available from Tesplane workers (subprocesses);
+* _can be intercepted_ indicates that this event can be intercepted and, accordingly, changed.
+
+Next come:
+* a description of the circumstances under which the event is triggered;
+* a snippet with code showing how to subscribe to it;
+* event handler parameters;
+* and optionally, examples of using this event in a plugin or plugins.
+
+## Events scheme {#events-scheme}
+
+TBD
+
+### Events scheme description {#events-scheme-description}
+
+Testplane can be launched either via the [CLI (command line)](../cli.md) or via its API: from a script using the `run` command.
+
+After launching, Testplane loads all plugins and proceeds to CLI parsing if it was launched via the CLI, or directly to the initialization stage if it was launched via the API.
+
+#### CLI Parsing {#cli-parsing}
```

Comment:
Perhaps making this an ordered list would improve the structure, but it's not critical to me (e.g. 1. CLI Parsing, 2. Initialization, 3. Reading Tests.